docs/cli/user-ingester-command.md

```diff
@@ -80,20 +80,20 @@ assignment creation for each user.
 
 Native ingesting users from a local CSV file:
```

Comment:
Same here, we could remove the Native from these sentences.

Reply:
Good catch, fixed